package.json

```diff
@@ -67,7 +67,7 @@
     "webpack": "1.12.13"
   },
   "peerDependencies": {
-    "cnvs": "1.1.3",
+    "cnvs": "~1.1.4",
```

Comment:
@mlunoe and I spoke offline about this and decided to go with `~1.1.x` for the `peerDependency`.